### DISCUSSION TITLE
Hello!

### DISCUSSION THREAD

**Discussion by Draglorr** [Doenerstyle/Invasion-Mod#3] [General]
I don't know if you'll see this or not, but i really wanted to try contacting you otherwise. I absolutely love the fact that i have _**FINALLY**_ found someone else who loves Invasion Mod (not Hostile worlds) as well!!!! It is my favorite mod of all time and I just want to say that I wholeheartedly support you! I wish you good luck and I would love to see this mod in 1.12.

Good luck and please, keep doing God's work! You are not alone in your love for this mod.

**Comment by Doenerstyle** [comment]
Hello! Yes, I do see this. :P

Yes, it is very hard to find anything useful about the Invasion Mod and almost feels like people forgot about it.
I was puzzled that nobody really picked it up or at least updated it, especially when considering some really stupid, easy-to-fix bugs that were present in the latest version.
But I found the original source code on GitHub and decided to update it myself to fix the things that bothered me.

As noted in the README, there are some people who started porting the mod to more recent versions, but none of them actually released anything and the latest (unreleased) 1.12.2 version was unplayable, sadly. (I didn't test the others)
I'd love to see this mod updated to 1.12, too, although I myself am mainly interested in 1.7.10. It **really** is saddening to see such a great mod just being left to rot, it has so much potential!

Thank you for the kind words and for sharing my love for this mod! If there is anything about it that bothers you or if you have a suggestion to make, please open an issue here or add me on Discord (name is on my profile) :)

**Reply by Draglorr** [reply]
I know right?? Actually for the longest time I could not find anywhere to download it as the original page went down which left me incredibly sad. After a very long time of searching, i did finally find a shady site hosting it. So instead of just letting it download and leaving it at that, I uploaded the mod to PlanetMinecraft myself so that there would always be a stable download place for anyone who wanted. 

Surprisingly, now whenever I look up Invasion Mod, basically the only place I could ever find a download was... the place I uploaded the mod. It shows up first, and is really the only thing that shows up. Actually multiple people commented on my archive I uploaded thanking me for actually putting up a download since they couldn't find literally ANYWHERE else. So I guess I helped out a lot more than I thought! Just figured you would like to know. For all I know, besides here, it is now the only download place for this mod left.

Why are you mainly interested in 1.7.10 i have to ask? It is a severly outdated version missing a wide array of features and QoL improvements! And yeah, I have to agree it has so much potential. It is so unique, it makes me wonder why it was left to rot as people make all these amazing defensive marvels, but I guess never want to test them?

Your welcome, you deserve the applause! I just recently updated the mod on Planet Minecraft with your version and then showered you in praise for it and linked here.  (Link: https://www.planetminecraft.com/mod/the-invasion-mod-4743554/)

**Reply by Doenerstyle** [reply]
Well, I have been able to find some mirrors and one person even reposted it on CurseForge (see README) - in French. But your repost looks more trustworthy and informative than any of them, for sure. Btw, you can check [archive.org](https://archive.org) when trying to access sites that are no longer accessible. I have linked the original forum thread of the Invasion Mod in the README of this fork.
I'm sure you've helped a lot of people! Also, thank you for linking this repo. :)

I mean, you could say the same about e. g. 1.12.2. To me, it's mostly the mods available for (only) 1.7.10, the combat update (1.9) that I disliked and just the overall feeling. I do understand why people prefer newer versions and don't blame them, though. To each their own!

Let's hope that the right person will stumble across this mod and actually update it. Would be very interesting to see what new things would be added!
